.github/workflows/mobile-deploy.yml

```
@@ -378,17 +414,20 @@ jobs:
 
       # android specific steps
       - name: Setup Java environment
+        if: github.event_name == 'workflow_dispatch' && github.event.inputs.platform != 'ios'
         uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)